projects/robots-farm/index.js

```diff
@@ -0,0 +1,23 @@
+const { sumTokens2 } = require('../helper/unwrapLPs');
+
+async function tvl(api) {
+  return sumTokens2({
+    api,
+    owners: ['0x51C84b14F63999604246Ec7De0e777500D0631D0'],
+    tokens: ['0x15De59489de5e7F240D72F787dC4a292b8199339','0x4200000000000000000000000000000000000006']
```

Comment:
thanks for the staking update, please also remove RBF from the core tvl function here